test/system/practice/submission_answer_test.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'application_system_test_case'
+
+class Practice::SubmissionAnswerTest < ApplicationSystemTestCase
+  setup do
+    @practice = practices(:practice1)
+  end
+
+  test 'student passed practice and product can show submission answer' do
+    user = User.find_by(login_name: 'kimura')
```

Comment:
[nits]書き方の好みの話になるのかもしれませんが、setupと書き方が異なるのでどちらかに統一した方がいいと感じました。
```suggestion
    user = users(:kimura)
```

Reply:
36d20fad6d216a5000d9fffb8cf7a7748ae9bd8b